### PR TITLE
Aligning structure of GMD to match D&I

### PR DESCRIPTION
**Pull request by klumb**
There is more to be done, but by aligning the structure of the work groups, it will make it easier to navigate and contribute to CHAOSS repositories.

### CODE REVIEW

File: 2_Growth-Maturity-Decline.md [hunk 1]
```diff
@@ -1,79 +0,0 @@
...
-[Pull Requests Open](activity-metrics/pull-requests-open.md) | Number of open pull requests.
-[Pull Requests Closed](activity-metrics/pull-requests-made-closed.md) | Number of closed pull requests.
-[Pull Request Comment Duration](activity-metrics/pull-requests-comment-duration.md) | The difference between the timestamp of the pull request creation date and the most recent comment on the pull request.
-[Pull Request Comment Diversity](activity-metrics/pull-requests-comment-diversity.md) | Number of each people discussing each pull request.
-[Pull Request Comments](activity-metrics/pull-request-comments.md) | Number of comments on each pull request. 
-
-
-[](activity-metrics/)
-
-## Community Growth
```
Comment (sgoggins) [inline 2_Growth-Maturity-Decline.md]: I think what you are getting at is providing a summary of the GMD metrics as a top level page, yes?

Comment (klumb) [reply 2_Growth-Maturity-Decline.md]: Agreed, these changes may be premature.

Comment (klumb) [reply 2_Growth-Maturity-Decline.md]: And yes, part of the pull request does do that.

**Comment by klumb** [comment]
Per GMD meeting:

- Adjust naming convention - remove measure and time
- Remove metrics that aren't being used
- Add file and folder structures for UseCases

Would like to merge this and then make further edits to finalize.

**Comment by sgoggins** [comment]
It seemed ok with us during the last GMD Call and we were going to make a final review, and if no comments merge.  Seems like this is ok, so I am going to merge the pull request.

**Pull request merged**